bin/flow.php

```diff
@@ -16,7 +16,7 @@
 use Symfony\Component\Console\Output\OutputInterface;
 use Symfony\Component\Console\Style\SymfonyStyle;
 
-if ('' !== \Phar::running(false)) {
+if ('' !== Phar::running(false)) {
```

Comment:
I mean, I have no idea why CS Fixer is doing that but at this point I don't care, I spent way too much time trying to understand what/why cs fixer is doing something, it become extremely unstable recently, I might need to lock it to some specific version.